src/Sarif/Notes.cs

```diff
@@ -43,9 +22,9 @@ public static void LogFileSkippedDueToSize(IAnalysisContext context, string skip
             // '{0}' was skipped as its size ({1} kilobytes) exceeds the currently configured threshold ({2} kilobytes).
             context.Logger.LogConfigurationNotification(
                 Errors.CreateNotification(
-                    context.TargetUri,
+                    new Uri(skippedFile, UriKind.RelativeOrAbsolute),
                     Msg002_FileSkippedDueToSize,
-                    ruleId: Msg002_FileSkippedDueToSize,
+                    ruleId: null,
```

Comment:
>null [](http://example.com/codeflow?start=28&length=4)

oops, this is an associated rule id, not the notification id, and should have been null.